BGL/doc/BGL/NamedParameters.txt

```diff
@@ -81,6 +81,34 @@ being marked or not.\n
 <b>Default:</b> a default property map where no edge is constrained
 \cgalNPEnd
 
+\cgalNPBegin{METIS_options} \anchor BGL_METIS_options
+is a parameter used in `partition()` to pass options to the METIS mesh
+partitioner. The many options of METIS are not described here. Instead, users
+should refer to METIS' <a href="http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/manual.pdf">documentation</a>.\n
+<b>Type:</b> an array of size `METIS_NOPTIONS` with value type `idx_t`
+            (an integer type defined by METIS). \n
+<b>Default:</b> an array of size `METIS_NOPTIONS` with value type `idx_t`,
+                initialized using the function `METIS_SetDefaultOptions()`.
+\cgalNPEnd
+
+\cgalNPBegin{vertex_partition_id_map} \anchor BGL_vertex_partition_id_map
+is the property map storing for each vertex of the mesh the id of the subpart
+of the partition that has been assigned to this vertex.\n
+<b>Type:</b> a class model of `ReadWritePropertyMap` with
+`boost::graph_traits<PolygonMesh>::%vertex_descriptor` as key type and
+`int` as value type. \n
+<b>Default:</b> a default property map.
```

Comment:
FWIU, default is none.

Reply:
Should I write it, though ? It doesn't matter what I'm using internally and I thought it'd leave me more room to wiggle and change the code later (say returning those default maps somehow) if I say that I'm using a default map rather than saying nothing is done.

Reply:
No it is on purpose that there are no default (since you can select which map is to be filled). I would simply document that there is no default.